ldclient.go

```diff
@@ -204,6 +204,48 @@ func (client *LDClient) Flush() {
 	client.eventProcessor.flush()
 }
 
+// Returns a map from feature flag keys to values for
+// a given user. If the result of the flag's evaluation would
+// result in the default value, `nil` will be returned. This method
+// does not send analytics events back to LaunchDarkly
+func (client *LDClient) AllFlags(user User) map[string]interface{} {
+	if client.IsOffline() {
+		client.config.Logger.Println("WARN: Called AllFlags in offline mode. Returning nil map")
+		return nil
+	}
+
+	if !client.Initialized() {
+		client.config.Logger.Println("WARN: Called AllFlags before client initialization. Returning nil map")
+		return nil
+	}
+
+	if user.Key == nil {
```

Comment:
As discussed-- we can't do that, since it would be an incompatible change in the semantics of v1.